examples/reminders/lib/feature_splash/splash_page.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:provider/provider.dart';
+
+import '../app_extensions.dart';
+import '../base/common_blocs/firebase_bloc.dart';
+
+class SplashPage extends StatefulWidget {
+  const SplashPage({Key? key}) : super(key: key);
+
+  @override
+  State<SplashPage> createState() => _SplashPageState();
+}
+
+class _SplashPageState extends State<SplashPage> {
+  static const _logoPath = 'assets/images/prime_logo.jpeg';
+
+  @override
+  void initState() {
+    super.initState();
+    startSplashTimer();
+  }
+
+  Future<Timer> startSplashTimer() async {
+    var duration = const Duration(seconds: 1);
+    return Timer(duration, _checkIfUserIsLoggedIn);
+  }
+
+  void _checkIfUserIsLoggedIn() {
+    context.read<FirebaseBlocType>().events.checkIfUserIsLoggedIn();
+  }
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        body: Center(
+          child: SizedBox(
+            height: 200,
+            child: Column(
+              children: [
+                _checkIsUserLoggedIn(),
```

Comment:
@StanislavPrime, be aware that according to the best practices the widget (in this case `RxBlocListener`) should be placed directly in the build method, while its callback (in this case the `listener`) should be extracted as a private method.

Reply:
I have placed the `RxBlocListener` in the build method and extracted the `listener` callback as a private method.